src/Client.ts

```diff
@@ -0,0 +1,119 @@
+// tslint:disable-next-line import-name no-require-imports
+import defaultsDeep = require('lodash.defaultsdeep');
```

Comment:
Not really, considering to ship my own implementation, but this is a significant step up from needing all of lodash 😛